### PR TITLE
Refactored alert component

### PR DESCRIPTION
**Pull request by bharati-21**
- Removed padding/ margin from styles and added to HTML as utility classes
- Edited basic styles for the alert components - added border to the left.
- Edited close icon as a button
- refactor sidebar links
- Removed alert component from docs/index.html and linked the alert component to docs/alert/index.html

### CODE REVIEW

**Review by AtulHande03a — APPROVED** [review]
nice use of variables for everything !!

File: docs/alert/index.html [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+                    </li>
+                    <li class="navlink">
+                        <a href="https://github.com/bharati-21/elixir-ui" target="_blank" class="link icon-link"><i class="fab fa-github"></i></a>
+                    </li>
+                </ul>
+                            
+            </nav>
+            <aside class="sidebar sidebar--close flex--col">
+                <div class="sidebar__header flex--row flex--align-center">
+                    <a href="/" class="brand flex--row flex--align-center flex--justify-center">
```
Comment (yashpatel1797) [inline docs/alert/index.html:44]: I don't have much knowledge in naming convention but sometimes I see you used '--' or '__' or '-' so if it is the same naming convention then fine or you can try to change this.

**Review by yashpatel1797 — APPROVED** [review]
Code looks good to me.

File: docs/alert/index.html (hunk 1 above)
Comment (bharati-21) [reply docs/alert/index.html:44]: Yes, the same naming convention. It's called BEM. Block element modifier. 
E.g. sidebar is the container and the block header resides within the sidebar. hence `sidebar__header`. And as for --close, it refers to a changed state for the sidebar, which is the modifier.

Hope this helps.

**Comment by bharati-21** [comment]
> Code looks good to me.

Thank you for your review 👍🏽

**Comment by bharati-21** [comment]
> nice use of variables for everything !!

Thank you!

**Pull request merged**